app/main.py

```diff
@@ -1,34 +1,77 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def init(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
+
+    def str(self) -> None:
```

Comment:
The method `str` should be `__str__` to properly override the string representation of the `Deck` class.